bin/crew

```diff
@@ -1736,10 +1736,6 @@ def upload(pkg_name = nil)
           system "sed -i 's/#{old_sha256}/#{new_sha256}/g' #{pkg_file}"
         end
       end
-      # Use rubocop to sanitize package file, and let errors get flagged.
-      system 'yes | crew install ruby_rubocop' unless @device[:installed_packages].any? { |elem| elem[:name] == 'ruby_rubocop' }
-      puts "Using rubocop to sanitize #{pkg_file} .".orange
-      system "rubocop -c #{File.join(CREW_LOCAL_REPO_ROOT, '.rubocop.yml')} -A #{pkg_file}", exception: true
```

Comment:
This is not a bad thing.

Reply:
We shouldn't be doing this in `crew`-- odds are this package file has been commited to a repo for us to get here, so it'll have been linted. If it hasn't been commited to a repo yet, it'll get flagged when it is.

Reply:
Slow on the uptake here. Do the pre-commit hooks allow for `rubocop -c .rubocop -A #{pkg_file}` to happen in some other manner before the commit happens?

I don't care how it happens. I just want it to happen.

Reply:
Yup! That's exactly what they do.

Reply:
You'll have to set them up
locally first by runnng pre-commit install, btw.